src/test/java/net/jodah/failsafe/issues/Issue52Test.java

```diff
@@ -45,7 +46,7 @@ public void shouldCancelExecutionViaFuture() throws Throwable {
     Thread.sleep(100);
     proxyFuture.cancel(true);
 
-    assertNull(proxyFuture.get());
```

Comment:
@jhalterman this test does not pass on master. Per the `.get()` javadoc, I think the correct result is meant to be that this throws rather than returning `null`. That's also how it currently behaves.